docs/reference/server-config.md

```diff
@@ -1,70 +1,118 @@
 <!--[metadata]>
 +++
-title = "Server Configuration File"
-description = "Specifies the configuration file for Notary Server"
-keywords = ["docker, notary, notary-server, configuration"]
+title = "Server Configuration"
+description = "Configuring the notary client, server and signer."
+keywords = ["docker, notary, notary-client, notary-server, notary server, notary-signer, notary signer"]
 [menu.main]
-parent="mn_notary"
-weight=80
+parent="mn_notary_config"
 +++
 <![end-metadata]-->
 
-# Notary Server Configuration File
+# Notary server configuration file
+
+This document is for power users of the [notary client](../advanced_usage.md),
+or for those who are [running their own service](../running_a_service.md) who
+want to facilitate CLI interaction or specify custom options.
+
+For full specific configuration information, see the [common configuration
+file](common-configs.md) for the Notary server and signer.
+
+## Overview
+
+A configuration file is required by Notary server, and the path to the
+configuration file must be specified using the `-config` option on the command
+line.
+
+The configuration file consists of the following sections:
+
+<table>
```

Comment:
Same answer as above:  I just couldn't figure out how to link the code parts into the different sections - this just let you skip down immediately.